resources/views/themes/tailwind/payments/paddle/settings/partials/invoices.blade.php

```diff
@@ -0,0 +1,54 @@
+<div class="p-8">
+
+    @subscriber
+    @php
+        $subscription = new \Wave\Http\Controllers\SubscriptionController;
```

Comment:
can we perform this call in controller that renders the view and just return data?
Performing direct php code in views is not considered as good practice

Reply:
We have this functionality because of setting section architecture in wave template.
![Screenshot from 2023-05-17 14-14-45](https://github.com/sapientpro/wave-stripe/assets/133088725/7fda095e-1797-4d4e-aa66-747c359a7c71)
We could add conditions section == 'invoices' & payment.vendor to get invoices but it will make the code more complicated as we chould extend passing optional variable in blades also.

we could try to move invoices to subscriptionTrait and use in User alike we do in stripe.settings.invoices blade:(auth()->user()->invoices())